packages/main/cypress/specs/base/InvisibleMessage.cy.ts

```diff
@@ -0,0 +1,33 @@
+import announce from "@ui5/webcomponents-base/dist/util/InvisibleMessage.js";
+import "../../../src/Button.js";
+import "../../../src/CheckBox.js";
```

Comment:
you need only one component and maybe we can render it.

Initial rendering test depends on framework boot that will come from some of the components. otherwise it is randomly failing.